mappings/net/minecraft/server/command/SpreadPlayersCommand.mapping

```diff
@@ -55,6 +56,9 @@ CLASS net/minecraft/class_3131 net/minecraft/server/command/SpreadPlayersCommand
 		ARG 15 respectTeams
 	METHOD method_29193 (Lcom/mojang/brigadier/context/CommandContext;)I
 		ARG 0 context
+	METHOD method_39991 (Ljava/lang/Object;Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 maxHeight
```

Comment:
Don't we use `height` as delta? The `execute` method already uses `maxY` for the parameter.
```suggestion
		ARG 0 maxY
```